DESCRIPTION

```diff
@@ -1,20 +1,24 @@
 Package: densityClust
 Type: Package
 Title: Clustering by Fast Search and Find of Density Peaks
-Version: 0.2.1
-Date: 2016-02-06
+Version: 0.3
+Date: 2017-04-17
 Authors@R: c(
-    person("Thomas Lin", "Pedersen", , "thomasp85@gmail.com", c("aut", "cre")),
-    person("Sean", "Hughes", , , c("aut"))
-    )
-Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>
-Description: An implementation of the clustering algorithm described by Alex
-    Rodriguez and Alessandro Laio (Science, 2014 vol. 344), along with tools to
-    inspect and visualize the results.
+  person("Thomas Lin", "Pedersen", , "thomasp85@gmail.com", c("aut", "cre")),
+  person("Sean", "Hughes", , , c("aut")),
+  person("Xiaojie", "Qiu", , "xqiu@uw.edu", c("aut")))
+Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>; Xiaojie Qiu <xqiu@uw.edu>
+Description: An improved implementation (based on k-nearest neighbors) of the density peak clustering algorithm, originally described by Alex Rodriguez and Alessandro Laio (Science, 2014 vol. 344). It can handle large datasets  (> 100, 000 samples) very efficiently. It was initially implemented by Thomas Lin Pedersen, with inputs from Sean Hughes and later improved by Xiaojie Qiu to handle large datasets with kNNs. This package is integrated in Monocle toolkit as a strategy to analyze unprecedentedly large single cell genomics datasets. 
 License: GPL (>= 2)
 Suggests:
     testthat
 LinkingTo: Rcpp
 Imports:
-    Rcpp
-RoxygenNote: 5.0.1
+    Rcpp, 
+    FNN, 
+    ggplot2,
+    ggrepel,
+    grDevices,
+    gridExtra,
+    RColorBrewer
+RoxygenNote: 6.0.1
```

Comment:
Can you make sure to only import functionality you use - I can't seem to find any use f ggplot2 and related packages..?

Reply:
I removed those unnecessary dependency